src/display.cpp

```diff
@@ -762,7 +762,7 @@ surface display::screenshot(bool map_screenshot)
 {
 	if (!map_screenshot) {
 		// Use make_neutral_surface() to copy surface content
-		return make_neutral_surface(screen_.getSurface());
+		return make_neutral_surface(CVideo::get_singleton().getSurface());
```

Comment:
What's the point of this change? Is `screen_` not guaranteed to be set or something?

Reply:
see comment below, you were just a little bit too fast.